src/insns/acperm_32bit.adoc

```diff
@@ -29,41 +29,38 @@ ACPERM performs the following operations:
 format shown in xref:acperm_bit_field[xrefstyle=short].
 . Calculate the bitwise AND of the bit field with the mask `rs2`.
 . If the AP and <<m_bit>> field in `cs1` could not have been produced by <<ACPERM>> then
-clear all AP permissions and the <<m_bit>>, and skip the next step
+clear all AP permissions, and the <<m_bit>> to {CAP_MODE_VALUE}. Skip the next step.
 . Clear AP permissions as required to meet the rules below.
-. Encode the AP permissions for MXLEN=32 according to <<cap_perms_encoding32>>.
+. Encode the AP permissions for RV32 according to <<cap_perms_encoding32>>.
 . Copy `cs1` to `cd`, and update the AP and SDP fields with the newly calculated versions.
 . Set `cd.tag=0` if `cs1` is sealed or if any reserved fields of `cs1` are set.
-+
-Some combinations of permissions cannot be encoded for MXLEN=32, and are not useful when MXLEN=64.
-These cases are defined to return useful minimal sets of permissions, which may be no permissions.
-+
-NOTE: Future extensions may allow more combinations of permissions, especially for MXLEN=64.
```

Comment:
Please keep something along these lines in the specification so that CHERIoT's permission semilattice could be a valid extension of Zcheripurecap.

Reply:
Agreed, I'd suggest keeping this note.